### DISCUSSION TITLE
【分享】获得菲林的快捷方法

### DISCUSSION THREAD

**Discussion by gaojiaobei** [share121/inter-knot#1667] [其他]
如果你是富裕的绳匠，那么你可以划走了，因为你不需要这种获得大量菲林的方法
而不是那么富裕的绳匠看过来，我找到了获取大量菲林的方法，并且不需要付出什么
[点我观看快速获取菲林的方法！快看速删！](https://www.bilibili.com/video/BV1im42137GG)

**Comment by Philia093-T** [comment]
![356807379-fbef40e8-9200-4997-97b5-cdf14295494b](https://github.com/user-attachments/assets/faa282ee-906f-40a9-874b-5d63a2beec45)

**Comment by VacuolePaoo** [comment]
markdown语法需要使用英文括号

**Comment by VacuolePaoo** [comment]
帮你改好了

**Reply by gaojiaobei** [reply]
> 帮你改好了

谢谢大佬🥰🥰

**Comment by gaojiaobei** [comment]
谢谢大佬 @VacuolePaoo 🥰🥰
